### DISCUSSION TITLE
Change Output Filename (it's too long)

### DISCUSSION THREAD

**Discussion by joebinglab** [Dadangdut33/Speech-Translate#42] [General]
I don't know if there's already a setting (that I can't find) or if it's possibility that this can go on the wishlist... but some of my filenames are already extremely long (due to downloading the exact title of the video) and the output always adds a timestamp at the front of the file and Mp4_Transcribed to the end. 

Example: 
**2023-08-14 14_51_32_294215** _My_Really_Long_Radio_-_Speaker_Host__-_The_Really_Long_Title_Of_The_Video_-_14_August_2023-[YouTubeVideoCode]_**.mp4_transcribed**.srt

Because the filename is so long, I often rename it to remove the timestamp and extra information at the end, but when it's too big, I can't even rename it (windows won't let me).

Is there some setting or can there be in future, where we can choose 'not' to add the timestamp, and not to add Mp4_transcribed to the start/end of the file?

I hope this makes sense :)

I love this program, and use it every single day :)

**Comment by Dadangdut33** [comment]
Thanks for the post and suggestion. What you said is correct it's way too long xD

i will add an option to format the filename of the output in the next release. Which will happen as soon as possible

**Reply by joebinglab** [reply]
Awesome, thank you so much :)
